perun-base/src/main/resources/perun-roles.yml

```diff
@@ -2122,13 +2166,17 @@ perun_policies:
       - VOADMIN: Vo
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Group
 
   addGroupsToAutoRegistration_List<Group>_policy:
     policy_roles:
       - GROUPADMIN: Group
       - VOADMIN: Vo
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Group
```

Comment:
in both of the AutoRegistration methods, we want to check Vo as well, since it affects the registration process for the Vos of the groups.